src/mcp_mt5/main.py

```
@@ -0,0 +1,1107 @@
+        flags: Type of requested ticks
+        
+    Returns:
+        List[Dict[str, Any]]: List of ticks.
+    """
+    # Convert datetime to timestamp in milliseconds
+    date_from_timestamp = int(date_from.timestamp() * 1000)
+    
+    ticks = mt5.copy_ticks_from(symbol, date_from_timestamp, count, flags)
+    if ticks is None:
+        logger.error(f"Failed to copy ticks for {symbol} from date {date_from}, error code: {mt5.last_error()}")
+        raise ValueError(f"Failed to copy ticks for {symbol} from date {date_from}")
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)